slackroll

```diff
@@ -970,6 +974,10 @@ def choose_option(option_list): # Prompt user and return option index
 
 def choose_pkg(pkg_list): # Returns chosen package or None
     options = [(str(x + 1), pkg_list[x].fullname) for x in xrange(len(pkg_list))]
+
+    if slackroll_batch_mode:
```

Comment:
Put this "if" block before creating the options list above, which would not be used.

Reply:
Addressed